nixos/modules/services/networking/xrdp.nix

```
@@ -101,85 +140,130 @@ in
         type = types.path;
         default = confDir;
         defaultText = literalMD "generated from configuration";
-        description = lib.mdDoc "The location of the config files for xrdp.";
+        description = lib.mdDoc ''
+          The location of the config files for xrdp that will be liked to
+          /etc/xrdp.
+        '';
+      };
+
+      #        Implementation Notes
+      # this approach is needed because if the whole
+      # folder is passed through then it become immutable,
+      # and /etc/xrdp is also used to store the secrets
+      # generated at first run
```

```suggestion
      #        Implementation Notes
      # this approach is needed because if the whole folder is passed through then it become immutable
      # and /etc/xrdp is also used to store the secrets generated at first run
```
so short lines are quite hard to read for some people

---

I found a way to drop the workaround, thank you @thiagokokada.